src/test/java/net/sf/jabref/logic/importer/fileformat/BibtexParserTest.java

```diff
@@ -236,6 +237,32 @@ public void parseRecognizesEntryWithUnknownType() throws IOException {
     }
 
     @Test
+    public void parseReallyUnknownType() throws Exception {
+        String bibtexEntry = "@ReallyUnknownType{test," + OS.NEWLINE +
+                " Comment                  = {testentry}" + OS.NEWLINE +
+                "}";
+
+        Collection<BibEntry> entries = new BibtexParser(importFormatPreferences).parseEntries(bibtexEntry);
+        BibEntry entry = entries.iterator().next();
+
+        String resourceName = "/testbib/reallyunknowntype.bib";
```

Comment:
Yes better compare it directly against a hand-created bibEntry (copy from above). 
Then you can use `assertEquals(singletonlist of expected entry, entries)`.

Apart from this last remark (sorry for being pedantic) this looks good to me and can be merged.